kubernetes/circuit-test-job.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: circuits-job
+spec:
+  ttlSecondsAfterFinished: 60  # Delete automatically 1 minute after completion
+  template:
+    spec:
+      nodeSelector:
+        cloud.google.com/machine-family: c3
+      containers:
+        - name: circuits-runner
+          image: wshino/circuits-runner:test5
```

Comment:
What is this image exactly?

Reply:
This is the container image I created using the method described in the README
This image must be pushed to a location where our GKE can access it.
In the future, it would be better to create an official zkemail docker registry and push to it or use our GCP's artifact registry